tools/utils.py

```diff
@@ -254,3 +254,14 @@ def remove_content(folder):
     import os, glob
     for file_name in glob.glob(os.path.join(folder, '*')):
         os.remove(file_name)
+
+## String manipulation utils
+def find_between(s, first, last):
```

Comment:
thanks for the suggestion, i modified the function

Reply:
There was one more thing that I forgot about. You used the logging module directly to print the error message instead of the slurmy logger. Also the exception handling didn't make sense anymore with the regex changes. I took the liberty to just quickly change it myself ;).